### ISSUE TITLE
Formatting inserts empty lines in specific case

### ISSUE THREAD

**Issue by bveldkamp** [pgadmin-org/pgadmin4#6785]
**Describe the bug**
I have a specific case where formatting the sql in a query window inserts an empty line inside a function between function arguments.  The 2nd argument is also moved too much to the right. 
I have never changed any formatting preferences, those are all default

**To Reproduce**
1. Paste this sql:
```
SELECT W.OBJECTID,
	G.SHAPE
FROM POINTS W
LEFT JOIN POLYGONS G ON ST_CONTAINS(G.SHAPE, W.SHAPE)
```
2. Format the SQL (Ctrl+Shift+K)
3. The result looks like this (scrolll to the right):
```
SELECT W.OBJECTID,
	G.SHAPE
FROM POINTS W
LEFT JOIN POLYGONS G ON ST_CONTAINS(G.SHAPE,

																									W.SHAPE)
```
 
**Expected behavior**
Code should be formatted as:
```
SELECT W.OBJECTID,
	G.SHAPE
FROM POINTS W
LEFT JOIN POLYGONS G ON ST_CONTAINS(G.SHAPE, W.SHAPE)
```
or maybe line up the arguments on separate lines:
```
SELECT W.OBJECTID,
	G.SHAPE
FROM POINTS W
LEFT JOIN POLYGONS G ON ST_CONTAINS(G.SHAPE, 
                                    W.SHAPE)
```

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Version: 7.6
 - Mode: Desktop

**Comment by anilsahoo20** [comment]
Related to issue #5083.

**Comment by pravesh-sharma** [comment]
This will be fixed with #5083

**Comment by anilsahoo20** [comment]
Tested and verified on candidate build: https://developer.pgadmin.org/builds/2024-02-05-1/
Environment: Debian 11
